packages/go_router/lib/src/route.dart

```diff
@@ -227,3 +145,55 @@ class GoRoute {
   ) =>
       const SizedBox.shrink();
 }
+
+/// A route that displays a UI shell around the matching child route.
```

Comment:
I'm guessing most folks that use `ShellRoute` will want to use `NestedStackRoute`. Should this comment mention `NestedStackRoute`?